modules/identity/src/Volo.Abp.Identity.Domain/Volo/Abp/Identity/IdentityUserManager.cs

```diff
@@ -285,4 +289,43 @@ public virtual async Task ResetRecoveryCodesAsync(IdentityUser user)
 
         await identityUserStore.SetTokenAsync(user, await identityUserStore.GetInternalLoginProviderAsync(), await identityUserStore.GetRecoveryCodeTokenNameAsync(), string.Empty, CancellationToken);
     }
+
+    public async override Task<IdentityResult> SetEmailAsync(IdentityUser user, string email)
+    {
+        var oldMail = user.Email;
+        
+        var result = await base.SetEmailAsync(user, email);
+        
+        result.CheckErrors();
+        
+        await DistributedEventBus.PublishAsync(
```

Comment:
We can consider checking the `oldValue != currentValue` and then publish the event.